packages/lib/src/prod/expose-production.ts

```diff
@@ -131,7 +131,7 @@ export function prodExposePlugin(
           res.set(filename, bundle[name])
           return res
         }, new Map())
-        item.code = item.code.replace(new RegExp(`'${DYNAMIC_LOADING_CSS_PREFIX}[^']+'`, 'g'), str => {
+        item.code = item.code.replace(new RegExp(`(["'])${DYNAMIC_LOADING_CSS_PREFIX}.*?\\1`, 'g'), str => {
```

Comment:
This is a good use case for regular group matching. Well done!!!

Reply:
thanks!